### ISSUE TITLE
Incorrect model renaming `prisma.transaction.create` -> `prisma.$transaction.create`

### ISSUE THREAD

**Issue by erawk** [prisma/codemods#5]
Thanks for the `codemods` tool. It worked mostly great for us. We came across three errors that required some fixes, and I thought you should know.

I can file separate issues for each if required.

1. `prisma.transaction.create` or `prisma.transaction.aggreggate` incorrectly converted to `prisma.$transaction.create`. We are an odd duck, because we have a table named `transaction`. 

Perhaps the regex or filter used should check to see if the char after the keyword is a `(`? That is, `prisma.transaction([sql1, sql2])` should be converted to `prisma.$transaction([sql1, sql2])`, but not `prisma.transaction.create`. 

2. As we are transitioning from TypeORM to Prisma, we still have some code that stands on two chairs. Using named imports from `@prisma/client` had renaming issues:
```typescript
import { User as PrismaUser } from '@prisma/client'
import { User } from '../users/UserModel'

export type UserLike = User | PrismaUser // TypeORM or Prisma user
```

Was changed to a broken version:
```typescript
import { Prisma } from '@prisma/client'
import { User } from '../users/UserModel'

export type UserLike = User | Prisma.User // Broken, as Prisma has no export User
```

In this case, we didn't need the `import` updated at all.

3. Non-`prettier` compatible semi-colon usage. Unnecessarily changed to add the semi-colon on a completely non-Prisma related file.

```diff
-  ].join('?')
+  ].join('?');
}
```

**Comment by williamluke4** [comment]
@erawk Thank you for your awesome write-up ❤️
1/2 These should be fixed in 0.6.0 (Could you please check :) 
3. Unfortunately this is a stale recast issue https://github.com/benjamn/recast/issues/226

**Comment by williamluke4** [comment]
Feel free to reopen if it is not fixed
